src/components/BucketListProfile.tsx

```diff
@@ -0,0 +1,121 @@
+import format from "date-fns/format";
+import React, { ReactElement, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { getBuckets, selectBuckets } from "../store/buckets/bucketsSlice";
+import { selectCredentialProfiles } from "../store/settings/settingsSlice";
+import Spinner from "./Spinner";
+import { PageTitle } from "./typography";
+
+export const BucketListProfile = (): ReactElement => {
+  const authSettings = useSelector(selectCredentialProfiles);
+
+  const { buckets, error, loading } = useSelector(selectBuckets);
+  const dispatch = useDispatch();
+
+  //Get CredentialProfile AccessKey and SeceretAccessKey by Id
+  let paramId: string;
+  const params = new URLSearchParams(window.location.search);
+  for (const param of params) {
+    paramId = param[0];
+  }
+  let getCredential: any;
+  const authSettingCredential = authSettings?.map((cp) => {
+    if (paramId == cp.id) {
+      getCredential = cp.credentials;
+    }
+  });
```

Comment:
I'm not really sure what we doing with the url stuff here, but we should be using route parameters for the profiles.
Perhaps a format like this:
```
/<profile-type>/<profile>/<bucket>/<folder>/<folder>
```
So as an example:
```
/storj-dcs/profile-one/bucket-one/folder-one/subfolder-one
```

Reply:
We might want to prefix `/profile` to the beginning of the routes, as well, so we can differentiate that base url from other urls that are at the root, such as `/settings`, etc...